node/utils/simulation.ts

```diff
@@ -105,25 +105,29 @@ export const simulateItem = async (
     country,
   })
 
-  const simulation = await checkout.simulation(...simulationPayload)
+  try {
+    const simulation = await checkout.simulation(...simulationPayload)
 
-  const {
-    items: [item],
-  } = simulation
+    const {
+      items: [item],
+    } = simulation
 
-  let itemInfo = {
-    price: 0,
-    available: false,
-  }
+    let itemInfo = {
+      price: 0,
+      available: false,
+    }
 
-  if (isSkuAvailable(item)) {
-    const { price, listPrice, unitMultiplier } = item
+    if (isSkuAvailable(item)) {
+      const { price, listPrice, unitMultiplier } = item
 
-    itemInfo = {
-      price: (Math.max(price, listPrice) * unitMultiplier) / 100,
-      available: true,
+      itemInfo = {
+        price: (Math.max(price, listPrice) * unitMultiplier) / 100,
+        available: true,
+      }
     }
-  }
 
-  return itemInfo
+    return itemInfo
+  } catch (error) {
```

Comment:
Isn't the error important to debug what caused the simulation to fail? If so, we could return it and log it with the `warn` or bring the logger here.